### PR TITLE
changes made to bonding.py to read different mtu sizes

### PR DESCRIPTION
**Pull request by vaishnavibhat**
differnt yaml files are also added to the list to address
different scenario based testing for mode 0,1, and 4
Signed-off-by: Vaishnavi Bhat <vaishnavi@linux.vnet.ibm.com>

### CODE REVIEW

File: io/net/bonding.py [hunk 1]
```diff
@@ -301,6 +303,8 @@ def bond_setup(self, arg1, arg2):
                 cmd = 'ip route add default via %s dev %s' % \
                     (self.gateway, self.bond_name)
                 process.system(cmd, shell=True, ignore_status=True)
+            if not HostInfo.set_mtu_host(self, self.bond_name, self.mtu):
```
Comment (narasimhan-v) [inline io/net/bonding.py]: @vaishnavibhat do we need to set mtu for peer bond interface also ?

**Comment by narasimhan-v** [comment]
@abdhaleegit @manvanthar Reviews please

**Comment by narasimhan-v** [comment]
@vaishnavibhat please take care of the review comment and Travis failure

**Comment by narasimhan-v** [comment]
@vaishnavibhat please take care of the one pending comment / question, and also attach job.log

**Comment by smruti77** [comment]
this PR handle in this #1579

**Pull request closed without merging**